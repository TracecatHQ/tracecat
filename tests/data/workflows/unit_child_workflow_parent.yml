title: Parent workflow
description: Test child workflow
entrypoint:
  ref: parent
inputs:
  data: [1, 2, 3, 4, 5, 6, 7]

actions:
  - ref: parent
    for_each: ${{ for var.x in INPUTS.data }}
    action: core.workflow.execute
    args:
      workflow_id: <<child_workflow_id>>
      loop_strategy: parallel
      fail_strategy: isolated
      batch_size: 3
      trigger_inputs:
        number: ${{ var.x }}
