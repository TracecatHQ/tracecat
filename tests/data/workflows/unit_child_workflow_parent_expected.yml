ACTIONS:
  parent:
    result: [1001, 1002, 1003, 1004, 1005, 1006, 1007]
    result_typename: "list"

INPUTS:
  data: [1, 2, 3, 4, 5, 6, 7]
TRIGGER:
